RepresentsMe/ViewControllers/MapViewController.swift

```diff
@@ -78,7 +81,22 @@ class MapViewController: UIViewController,
     /// Hide the navigation bar when the view appears
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.setNavigationBarHidden(true, animated: animated)
+        switch reachType {
+        case .map:
+            navigationController?.setNavigationBarHidden(false, animated: animated)
+                listButton.image = UIImage.fontAwesomeIcon(
+                    name: .list,
+                    style: .solid,
+                    textColor: .blue,
+                    size: CGSize(width: 24, height: 24))
+                listButton.isEnabled = true
+                break
+        case .event, .settings:
```

Comment:
Change "case .event, .settings:" to "default:" so that we dont have to add more cases to this statement if we use the mapview elsewhere (like picking an address at sign up)